app/apps/addresses/models.py

```diff
@@ -101,4 +112,39 @@ def save(self, *args, **kwargs):
                 raise DistrictNotFoundError(
                     f"verblijfsobject_url: {verblijfsobject_url}, verblijfsobject: {verblijfsobject}"
                 )
+
+    def get_bag_nummeraanduiding_id(self):
+        try:
```

Comment:
Iets explicietere naming is beter denk ik.
Een get_ functie impliceert dat die de nummeraanduiding terug geeft.
Maar je doet dus eigelijk een set op grond van bag search.